activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/BaseBpmnXMLConverter.java

```diff
@@ -205,14 +205,14 @@ public void convertToXML(XMLStreamWriter xtw, BaseElement baseElement, BpmnModel
       xtw.writeEndElement();
     }
 
+    writeAdditionalChildElements(baseElement, model, xtw);
```

Comment:
This change is going to break multi-instanciated script tasks because it will move `<script>` element before `<multiInstanceLoopCharacteristics>`. I'd suggest stopping using the generic method `writeAdditionalChildElements` to write `incoming` and `outgoing` flow elements and create a new method `writeIncomingOutgoingFlowElements` for that. Then we have:
- writeIncomingOutgoingFlowElements
- writeMultiInstances
- writeAdditionalChildElements

Does it make sense?

Reply:
Changes are made